client/src/app/login/login.component.css

```diff
@@ -0,0 +1,85 @@
+* {
+    font-family: 'JetBrains Mono', monospace;
+}
+
+.login-form-container {
+        flex-direction: column;
+        justify-content: center;
+        text-align: center;
+        width: 434px;
+        margin: 15% auto;
+        
+    
+}
+.login-title {
+        text-transform: uppercase;
+        font-weight: 800;
+        font-size: 24px;
+        line-height: 32px;
+        margin: 27px auto;
+}
+.login-container {
+    
+        justify-content: center;
+        flex-direction: column;
+        
+        display: flex;
+        height: 100%;
+        width: 434px;
+        height: 318px;
+        border: 1px solid #000000;
+        border-radius: 10px;
+        background-color: #054A91;
+        
+    }
+    .image-border {
+        margin-top: 78px;
+        border: 2px solid #000000;
+        border-radius: 217px;  
+        padding-left: 1px;
+        padding-right: 1px;
+
+    }
+    .log-input-password, .log-input-username{
+        background-color: #F5F5F5;
+        border-radius: 10px;
+        width: 400px;
+        height: 50px;
+        padding: 5px;
+    }
+
+    .log-input-password{
+        margin-bottom: 42px;
+    }
+
+    .log-input-label {
+        padding-left: 20px;
+        font-style: normal;
+        font-weight: 700;
```

Comment:
I think 700 font is not imported in main css. I guess it will be ok when you merge it

Reply:
And font size must be 18px